app/controllers/project_database.py

```diff
@@ -196,6 +196,42 @@ def get(self, project_id, database_id):
         return dict(status='success', data=dict(database=json.loads(database_data))), 200
 
 
+class ProjectDatabaseStatusView(Resource):
```

Comment:
I also do not think we might need an endpoint to return db status.
We might return db status at the endpoint that shows info about a single db. `GET` method on `/databases/<database_id>`
What do you think?

Reply:
@Mubangizi you make a good point